### PR TITLE
Docs and co

### PR DESCRIPTION
**Pull request by maelle**
Would you be ok with submitting 1.0.4 to CRAN?

Changes made in this PR all are related to docs (local docs and pkgdown website). After merging you'd still need to run `travis::use_travis_deploy()` and to change the gh-pages settings to get a website, whose URL should be added in DESCRIPTION, repo settings, and in `_pkgdown.yml`.

- Added a pkgdown config file with a reference TOC and the R Consortium author info.
- Changed the version in DESCRIPTION and added the R Consortium as funder.
- Made NEWS and README top-level. NEWS.md was slightly changed to make it pkgdown-compatible (for the changelog to be generated, the section headers need to contain the package name)
- Added a package-level manual page.
- Copy-pasted the Travis config from `rhub` :-)
- Added an Rproj (ok, this has nothing to do with docs)

### CODE REVIEW

**Comment by maelle** [comment]
Should this repo be transferred to r-hub GitHub organization btw? This way the pkgdown website would have a r-hub.github.io address.

File: README.Rmd [hunk 1]
```diff
@@ -38,6 +45,23 @@ r_oldrel()
 r_versions()
 ```
 
+
+## Use cases in the wild
```
Comment (gaborcsardi) [inline README.Rmd]: Why do we want these? These links are bound to go out of date, no?

File: README.Rmd [hunk 2]
```diff
@@ -38,6 +45,23 @@ r_oldrel()
...
+* [`provisionr`](https://github.com/mrc-ide/provisionr) imports `rversions` and uses it in `check_r_version()`, a function that checks and coerces something into an R version.
+
+### In other tools
+
+- [r-appveyor web service uses `rversions` to determine what versions r-release and r-oldrel are](https://github.com/krlmlr/r-appveyor/blob/master/scripts/appveyor-tool.ps1#L77-L87).
+
+- R-hub uses it to do the same on [Solaris](https://github.com/r-hub/solarischeck/blob/d3edc6077f52081a53b2ed8b9ec40e17176fdb43/run.sh#L105), [macOS](https://github.com/r-hub/macoscheck/blob/846f87d1ad1c4b4aeb283a986e084913f203879f/run.sh#L160) and [Windows](https://github.com/r-hub/wincheck/blob/591ea8d9a28cd5044c063e88f46ef0507fa16fc7/run.ps1#L114).
+
+- [R-hub's Windows install script uses it as well, to decide what versions of R to install](https://github.com/r-hub/rhub-server/blob/master/jenkins/scripts/windows-2008-setup.ps1#L74-L79).
+
```
Comment (gaborcsardi) [inline README.Rmd]: Same here. I think these are ok to mention in a blog post, but we don't need them in a readme.

File: _pkgdown.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+authors:
+  R Consortium:
+    href: "https://www.r-consortium.org/"
+    html: "<img src='https://www.r-consortium.org/wp-content/uploads/sites/13/2016/09/RConsortium_Horizontal_Pantone.png' height=24>"
+
+
+reference:
+- title: Shortcuts
```
Comment (gaborcsardi) [inline _pkgdown.yml]: These are not really shortcuts, i.e. `r_versions()` does not give you the same information

**Review by gaborcsardi — COMMENTED** [review]
Thanks!

File: README.Rmd (hunk 1 above)
Comment (maelle) [reply README.Rmd]: right but it's interesting for people landing on the docs, and the README is .Rbuildignored :-)

File: README.Rmd (hunk 2 above)
Comment (maelle) [reply README.Rmd]: will remove the section then :-)

File: _pkgdown.yml (hunk 3 above)
Comment (maelle) [reply _pkgdown.yml]: how would you call them?

File: README.Rmd (hunk 1 above)
Comment (gaborcsardi) [reply README.Rmd]: we can un-ignore the readme, but I don't think we need "usage in the wild"

File: _pkgdown.yml [hunk 4]
```diff
@@ -5,7 +5,7 @@ authors:
 
 
 reference:
-- title: Shortcuts
+- title: Single R versions
```
Comment (gaborcsardi) [inline _pkgdown.yml]: "Version labels"?

Comment (gaborcsardi) [reply _pkgdown.yml]: I don't love this either, so up to you....

File: inst/README.md [hunk 5]
```diff
@@ -160,34 +160,6 @@ r_versions()
...
-  - R-hub uses it to do the same on
-    [Solaris](https://github.com/r-hub/solarischeck/blob/d3edc6077f52081a53b2ed8b9ec40e17176fdb43/run.sh#L105),
-    [macOS](https://github.com/r-hub/macoscheck/blob/846f87d1ad1c4b4aeb283a986e084913f203879f/run.sh#L160)
-    and
-    [Windows](https://github.com/r-hub/wincheck/blob/591ea8d9a28cd5044c063e88f46ef0507fa16fc7/run.ps1#L114).
-
-  - [R-hub’s Windows install script uses it as well, to decide what
-    versions of R to
-    install](https://github.com/r-hub/rhub-server/blob/master/jenkins/scripts/windows-2008-setup.ps1#L74-L79).
-
```
Comment (gaborcsardi) [inline inst/README.md]: Maybe we can just have some generic text that R-hub uses this to determine which version to use for R-release and R-oldrel?

**Comment by maelle** [comment]
Ready to merge? Will update the reference TOC as soon as I find a better idea!

**Comment by gaborcsardi** [comment]
just left another comment, sry.

File: inst/README.md (hunk 5 above)
Comment (maelle) [reply inst/README.md]: Added one sentence, not sure if it's a good sentence though (but I agree with the suggestion).

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**